remove 'Authorization Code Grant' as supported authentication (#12)
camaraproject/QualityOnDemand_PI1: README.md

- **Swagger Specification:** The API adheres to the provided Swagger specification (https://swagger.io/specification/v2/) to ensure consistency and facilitate client development.

> - **Authentication:** The API utilizes OAuth 2.0 for authentication. Two flows are supported:

Reviewer: ```suggestion
- **Authentication:** The API utilizes OAuth 2.0 for authentication. Currently the below flow is supported:
```
